### DISCUSSION TITLE
osu!lazer can't find the osu!stable file directory

### DISCUSSION THREAD

**Discussion by HydriZz** [ppy/osu#17471] [Q&A]
Hello, I just bought an external hard drive and I moved my osu!stable folder in it. I also changed location of my osu!lazer folder in that drive. I want to import everything of stable in lazer, but it seems like osu!lazer can't find the new location of my osu!stable folder and it creates a ''sub-osu!stable'' download. 

Here is a picture importing maps:
![osu_2022-03-26_19-59-24](https://user-images.githubusercontent.com/102440884/160260985-5fa4eca9-714e-4c29-acb2-47a445227546.jpg)
As you can see, it only imports the default maps when you start osu for the first time.
Is there a way to locate the osu!stable folder manually? 

Thanks in advance,
HydriZ

**Comment by smoogipoo** [comment]
I think you should be able to open osu!stable from your new directory and click "repair folder permissions" at the bottom of settings, then try importing again.
